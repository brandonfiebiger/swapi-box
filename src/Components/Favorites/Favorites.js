import React from 'react';

const Favorites = (props) => {

  return(
    <div>
      View Favorites {props.favorites.length}
    </div>
  )  
}

export default Favorites;