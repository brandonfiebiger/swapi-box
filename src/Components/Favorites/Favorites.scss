@import "variables.scss";

button {
  height: 3rem;
  margin-left: auto;
  border: 5px solid $button-border;
  border-radius: 10px;
  color: #f2f2f2;
  background-color: $button-color;
  text-shadow: 1.5px 1.5px $shadow-color;
  font-size: 1.5rem;
  outline: none;
  &:hover {
    border: 5px solid $button-color;
    text-shadow: 1px 1px $red-highlight;
    cursor: pointer;
  }
}

@media only screen and (max-width: 1225px) { 
  button {
    margin-left: 2rem;
  }
}

@media only screen and (max-width: 1045px) { 
  button {
    margin: 1rem auto;
  }
}