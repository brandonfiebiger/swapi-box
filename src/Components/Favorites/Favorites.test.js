import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, mount } from 'enzyme';

import Favorites from './Favorites';

describe('Favorites', () => {

  let wrapper;
  let favorites;

  beforeEach(() => {
    favorites = [];
    wrapper = shallow(<Favorites favorites={favorites} />);
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })

  it('should match snapshot if number of favorites changes', () => {
    favorites = ['Luke Skywalker'];
    expect(wrapper).toMatchSnapshot();    
  })

  it('should invoke a method called displayFavorites when clicked', () => {

  })
})