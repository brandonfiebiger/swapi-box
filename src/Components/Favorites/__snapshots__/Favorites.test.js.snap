// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorites should match snapshot if number of favorites changes 1`] = `undefined`;

exports[`Favorites should match the snapshot 1`] = `undefined`;
